Lesson14/Gokhan/script.js

```diff
@@ -0,0 +1,29 @@
+// 9. Declare a variable number6 and initialize it to 6. 
+//Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+
+const number6 = 6;
```

Comment:
Variable `number6` needs to be instantiated using `let` keyword here so that you can change its value on the next line.
```suggestion
let number6 = 6;
```